tests/InputFiles/ScalarAdvection/Sinusoid1D.yaml

```diff
@@ -0,0 +1,66 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: EvolveScalarAdvectionSinusoid1D
+# Check: parse;execute
+# ExpectedOutput:
+#   ScalarAdvectionSinusoid1DVolume0.h5
+#   ScalarAdvectionSinusoid1DReductions.h5
+
+Evolution:
+  InitialTime: 0.0
+  InitialTimeStep: 0.001
+  TimeStepper: RungeKutta3
+
+PhaseChangeAndTriggers:
+
+DomainCreator:
+  Interval:
+    LowerBound: [-1.0]
+    UpperBound: [1.0]
+    InitialRefinement: [1]
+    InitialGridPoints: [2]
+    TimeDependence: None
+    BoundaryConditions:
+      LowerBoundary: Periodic
+      UpperBoundary: Periodic
+
+SpatialDiscretization:
+  BoundaryCorrection:
+    Rusanov:
+  DiscontinuousGalerkin:
+    Formulation: StrongInertial
+    Quadrature: GaussLobatto
+
+AnalyticSolution:
+  Sinusoid:
+
+Limiter:
+  Minmod:
+    Type: LambdaPi1
+    # The optimal value of the TVB constant is problem-dependent.
+    # This test uses 0 to favor robustness over accuracy.
+    TvbConstant: 0.0
+    DisableForDebugging: false
+
+EventsAndTriggers:
+  ? Slabs:
+      Specified:
+        Values: [1000]
+  : - Completion
+  # ? Slabs:
```

Comment:
why are these commented out?

Reply:
@kidder If uncommented, the following error comes out
```
InputFiles.ScalarAdvection.Sinusoid1D.yaml.execute ...
***Failed  Error regular expression found in output. Regex=[ERROR]
```
when running `ctest` in local. Uncommented one works fine for running executables, but have no idea why this makes test fail..?

Reply:
Shall I just delete these lines? (I left these as comments, not deleted, just for convenience so that I do not need to type necessary options every time when I run executables after rebasing branches)